src/asana/helpers.py

```diff
@@ -84,6 +84,26 @@ def _task_status_from_pull_request(pull_request: PullRequest) -> str:
     else:
         return "Draft" if pull_request.is_draft() else "Open"
 
+def _review_status_from_pull_request(pull_request: PullRequest) -> Optional[str]:
+    approval_or_changes_requested_reviews = sorted(
+        (
+            review
+            for review in pull_request.reviews()
+            if review.is_approval_or_changes_requested()
+        ),
+        key=lambda r: r.submitted_at(),
+    )
+
+    if len(approval_or_changes_requested_reviews) == 0:
+        # We didn't find any reviews
+        return "Needs Review"
+
+    latest_review = approval_or_changes_requested_reviews[-1]
+    if latest_review.is_approval():
```

Comment:
Does this account for the state where one person requested changes and another person approves later?

SGTM currently waits for all reviewers that requested changes to approve before it determines that the PR has been "approved" instead of looking at the last review. You can checkout `pull_request.is_approved` logic

Reply:
I don't think this makes sense to wait for everyone to approve the PR to mark the PR as approved. 
I think it makes more sense to refer to the latest review left on the PR. What if user A requests changes that are not actually needed and user B follows up saying it's okay and approves the changes. If user B does agree with user A's comments, they'll just add in their additional comments or also request changes as well.

Thoughts?